lib/inspec/cached_fetcher.rb

```diff
@@ -8,7 +8,7 @@ class CachedFetcher
     attr_reader :cache, :target, :fetcher
     def initialize(target, cache)
       @target = target
-      @fetcher = Inspec::Fetcher.resolve(target)
+      @fetcher = Inspec::FetcherSystem.resolve(target)
```

Comment:
Is this PR on top of the other fetcher PR? It looks like it is based off of master.